src/app/shared/enum/descriptorType.enum.ts

```diff
@@ -13,8 +13,13 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-
+/**
+ * This is mainly used as an enum replacement in the GA4GH endpoints until Swagger provides the enum
```

Comment:
Hasn't this been in there for a while? 
https://github.com/ga4gh/tool-registry-service-schemas/blob/develop/src/main/resources/swagger/ga4gh-tool-discovery.yaml#L598

Reply:
going to do https://github.com/ga4gh/dockstore/issues/1628 right after